Makefile

```diff
@@ -46,18 +47,18 @@ endif
 .PHONY: docker
 
 ifndef VERSION
-VERSION := 9
+VERSION := 10
 endif
 
 docker_check:
 	@which docker > /dev/null || (echo "*** Docker appears to be missing. Please install docker.io in order to build OpenNetworkLinux." && exit 1)
 
 docker: docker_check
-	@docker/tools/onlbuilder -$(VERSION) --isolate --hostname onlbuilder$(VERSION) --pull --autobuild --non-interactive
```

Comment:
TODO: add a note about this
basically the image on Docker hub is too old, need to build a new one locally